### PR TITLE
JBQA-13276 introduce possibility to use CacheManager.

### PR DESCRIPTION
**Pull request by tremes**

### CODE REVIEW

File: src/main/java/jmeter/plugins/http2/sampler/HttpResponseAdapter.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    public static BasicHttpResponse wrapAsApacheHttpReponse(FullHttpResponse fullHttpResponse) {
+        String reasonPhrase = fullHttpResponse.status().reasonPhrase();
+        int code = fullHttpResponse.status().code();
+        ProtocolVersion version = new ProtocolVersion("HTTP", 2, 0);
+        BasicStatusLine syntheticLine = new BasicStatusLine(version, code, reasonPhrase);
+        BasicHttpResponse response = new BasicHttpResponse(syntheticLine);
+        for (Map.Entry<String, String> header : fullHttpResponse.headers()) {
+            response.addHeader(header.getKey(), header.getValue());
+        }
+        response.addHeader("Cache-Control","max-age=2592000");
```
Comment (tremes) [inline src/main/java/jmeter/plugins/http2/sampler/HttpResponseAdapter.java:25]: This could be probably configurable via some system property.

**Comment by tremes** [comment]
I also think we should update licenses in all files in this repository to include JBoss/Red Hat copyright. WDYT?

**Comment by rhatlapa** [comment]
This is contribution to upstream version of the HTTP/2 jmeter sampler which initial version wasn't done by us. Even though the original creator doesn't seem to update the original one or accept merge requests, and we have done quite extensive update to the plugin, I am not sure.

**Comment by rhatlapa** [comment]
The changes LGTM, merging.

**Pull request merged**